code/modules/spells/spell_types/undirected/list_target/convert_role/adept.dm

```diff
@@ -0,0 +1,18 @@
+/datum/action/cooldown/spell/undirected/list_target/convert_role/adept
+	name = "Recruit Adept"
+	button_icon_state = "recruit_guard"
+	new_role = "Adept"
+	recruitment_faction = "Inquisition"
+	recruitment_message = "You will serve Psydon's will, %RECRUIT!"
+	accept_message = "F-fine just don't kill me!"
+	refuse_message = "I FOLLOW MY GOD INTO DEATH!!!"
+
+/datum/action/cooldown/spell/undirected/list_target/convert_role/adept/can_convert(mob/living/carbon/human/cast_on)
+	. = ..()
+	if(!.)
+		return
+	//Can't convert devoted faithfuls
```

Comment:
I would rather we don't force that, some gods like Baotha or Zizo would have 0 issues with you preserving your own life even if it means working for the inquisition.